### DISCUSSION TITLE
alt+c and alt+shift+c cycles tabs - How to disable?

### DISCUSSION THREAD

**Discussion by yogapotato** [gdh1995/vimium-c#583] [Q&A] [answered]
When I enter alt+c, it goes one tab to left, and alt+shift+c goes one tab to the right. However, I can't find this option anywhere in the command listing, and when I try to unmap "c", it tells me the key is not mapped. Is this behaviour intended?

**Comment by gdh1995** [comment, marked as answer]
```
unmap <a-c>
unmap <a-s-c>
```
